scio-extra/src/main/scala/com/spotify/scio/extra/checkpoint/package.scala

```diff
@@ -52,8 +51,9 @@ package object checkpoint {
      * @param fn result of this arbitrary => [[com.spotify.scio.values.SCollection SCollection]]
      *           flow is what is checkpointed
      */
+    @deprecated("Checkpoint support is deprecated, " +
+      "use smaller workflows and orchestration framework instead", "0.8.0")
     def checkpoint[T: Coder](fileOrPath: String)(fn: => SCollection[T]): SCollection[T] = {
-      FileSystems.setDefaultPipelineOptions(self.options)
```

Comment:
was this remove intended? we can just leave as is since this is deprecated?

Reply:
Yes, we added this to `ScioContext` constructor so this work around is no longer needed.

Reply:
ahh yes right